templates/cloudfront-site.yaml

```diff
@@ -103,7 +103,9 @@ Resources:
     Type: AWS::CloudFront::OriginAccessControl
     Properties:
       OriginAccessControlConfig:
-        Name: !Sub 'oac-${AWS::StackName}'
+        # Get a unique ID for the OAC Config name.
+        # Name must be unique within account
+        Name: !Select [2, !Split ['/', !Ref 'AWS::StackId']]
```

Comment:
You might want to put some indication of what application owns the OAC so customers can tell at a glance in the console.

```yaml
Name: !Sub 
  - amzn-secure-static-site-${guid}
  - guid: !Select [2, !Split ['/', !Ref 'AWS::StackId']]
```

Reply:
Great idea!